pkg/events/event.go

```diff
@@ -51,4 +51,10 @@ const (
 	UnsupportedVerticaVersion       = "UnsupportedVerticaVersion"
 	ATConfPartiallyCopied           = "ATConfPartiallyCopied"
 	S3AuthParmsCopyFailed           = "S3AuthParmsCopyFailed"
+	UpgradeStart                    = "UpgradeStart"
+	UpgradeSucceeded                = "UpgradeSucceeded"
```

Comment:
How do we handle upgrade failure?

Reply:
Yes, we go into a requeue/retry loop.  It will keep trying to start the database.  It is up to the user to change the image back to something else to "rollback" the upgrade.  We have status conditions that you can use with `kubectl wait`, to know if the upgrade times out.